libs/community/langchain_community/cache.py

```diff
@@ -1103,6 +1130,7 @@ def __init__(
         )
 
     def lookup(self, prompt: str, llm_string: str) -> Optional[RETURN_VAL_TYPE]:
+        """Look up based on prompt and llm_string."""
```

Comment:
The docstring is inherited from the parent class. So no need to override it with one with less details.
```suggestion
```